lib/livebook/teams/requests.ex

```diff
@@ -227,6 +231,34 @@ defmodule Livebook.Teams.Requests do
     get("/api/v1/org/identity", %{access_token: access_token}, team)
   end
 
+  @doc """
+  Send a request to Livebook Team API to return a session using a deploy key.
+  """
+  @spec fetch_cli_session(map()) :: api_result()
+  def fetch_cli_session(config) do
+    post("/api/v1/cli/auth", %{}, config)
```

Comment:
We call this with `config = %Team{}`, do we actually want to send all the fields?

Reply:
To call this function, we don't really use the `%Team{}` struct, we send a map to fetch some data from Teams, so then we can build the `%Team{}` struct.